Code/EnginePlugins/VisualScriptPlugin/Runtime/VisualScript.h

```diff
@@ -246,4 +259,20 @@ class EZ_VISUALSCRIPTPLUGIN_DLL ezVisualScriptExecutionContext
   ezScriptCoroutine* m_pCurrentCoroutine = nullptr;
 };
 
+struct ezVisualScriptSendMessageMode
+{
+  using StorageType = ezUInt8;
+
+  enum Enum
+  {
+    Direct,
+    Recursive,
+    Event,
```

Comment:
I assume this is mainly "internal" ? Or would it make sense to add some comment here ?